oops/oops_test.go

```diff
@@ -476,10 +488,11 @@ testing.tRunner
 		},
 	}
 
-	re := regexp.MustCompile(`\.(go|s):\d+`)
 	for _, testcase := range testcases {
 		t.Run(testcase.Title, func(t *testing.T) {
-			actualVerbose := re.ReplaceAllString(fmt.Sprint(testcase.Error), `.$1:123`)
+			actualVerbose := fixLineNumbers(fmt.Sprint(testcase.Error))
+			actualVerbose, err := stripPathPrefix(actualVerbose)
+			assert.Empty(t, err)
```

Comment:
Could [assert.NoError](https://pkg.go.dev/github.com/stretchr/testify/assert#NoError) be a better fit here (and also further along in the code, too)?

```suggestion
			assert.NoError(t, err, "received unexpected error from stripPathPrefix")
```

Reply:
ah yeah agreed